app/assets/javascripts/gobierto_budgets/maps.js

```diff
@@ -16,6 +16,8 @@ $(document).on('turbolinks:load', function() {
     var mapPropertiesLat = window.mapSettings.centerLat || 40.416775
     var mapPropertiesLon = window.mapSettings.centerLon || -3.703790
     var mapPropertiesZoom = window.mapSettings.zoomLevel || 5
+    var mapPlaceScope = typeof window.placesScope != "undefined" && window.placesScope != null && window.placesScope.length != null && window.placesScope.length > 0
+    var urlTOPOJSON = mapPlaceScope ? "/cat-municipis.json" : "/municipalities_topojson.json"
```

Comment:
Could we make it an optional parameter? I'd prefer it to be an optional parameter in https://github.com/PopulateTools/gobierto-comparador-presupuestos/blob/master/app/views/gobierto_budgets/pages/map.html.erb#L121 

if no file is provided, it'll use municipalities_topojson.json